content/miq_dialogs/miq_provision_ibm_dialogs_template.yaml

```diff
@@ -61,39 +98,64 @@
     :hardware:
       :description: Profile
       :fields:
-        :number_of_sockets:
-          :values:
-            1: 1
-            2: 2
-            4: 4
-            8: 8
-          :description: Cores (CPUs)
+        :entitled_processors:
+          :description: Entitled CPUs
           :required: true
+          :required_method: :validate_entitled_processors
           :display: :edit
-          :default: 1
-          :data_type: :integer
+          :default: '0.25'
+          :data_type: :string
+          :min_length: 1
+          :max_length: 5
         :instance_type:
           :values:
-            1: "capped"
-            2: "uncapped"
-            3: "dedicated"
+            0: "capped"
+            1: "shared"
+            2: "dedicated"
           :description: Processor
           :required: true
           :display: :edit
-          :default: 1
+          :default: 0
+          :data_type: :integer
+        :sys_type:
+          :values_from:
+            :method: :allowed_sys_type
+          :description: Machine Type
+          :required: true
+          :display: :edit
+          :default: 0
+          :data_type: :integer
+        :storage_type:
+          :values_from:
+            :method: :allowed_storage_type
+          :description: Storage Type
+          :required: true
+          :display: :edit
+          :default: 0
           :data_type: :integer
         :vm_memory:
-          :values:
-            2:  2
-            4:  4
-            8:  8
-            16: 16
-            32: 32
-            64: 64
           :description: Memory (GB)
+          :required_method: :validate_regex
+          :required_regex: !ruby/regexp /^\s*[0-9]+\s*$/
+          :required: true
+          :default: "2"
+          :data_type: :string
+          :display: :edit
+        :cloud_volumes:
+          :values_from:
+            :method: :allowed_cloud_volumes
+          :description: Attach Volumes
+          :required: false
+          :display: :edit
+          :auto_select_single: false
+          :data_type: :string
+        :guest_access_key_pair:
+          :values_from:
+            :method: :allowed_guest_access_key_pairs
+          :description: Key Pair
```

Comment:
The web UI uses "SSH Key", but in this case maybe it's best to keep "Key Pair" to stay consistent with ManageIQ's label?
![image](https://user-images.githubusercontent.com/1637291/92963944-e543e780-f438-11ea-8f87-687bdaa251d6.png)

Reply:
we could keep "Key Pair" for now, it is easy to change anyway